tests/jvm/src/test/resources/example/CaseClasses.scala

```diff
@@ -0,0 +1,24 @@
+package example
```

Comment:
I think we don't actually need the whole new file to test it - better to add a new invocation testing the issues in one of the existing classes.

I would especially concentrate on the code that is shown in the issue - it will be easier to see what is fixed.